src/Test.UnitTests.Sarif/TestData/SarifCurrentToVersionOneVisitor/Inputs/ResultWithTwoPartialFingerprints.sarif

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json.schemastore.org/sarif-2.1.0-rtm.3",
+  "version": "2.1.0",
+  "runs": [
```

Comment:
>runs [](start = 3, length = 4)

again, wouldn't have created individual files for these, would minimally have tended to merge the cases with one or two partial fingerprints. not sure how it helps to have them factored differently. the value of knowing the exact condition that failed (because it is encoded in the file name) is not worth the direction we're headed towards: analyzing hundreds of files in order to validate lots of detailed runtime conditions.